### PR TITLE
Feature nxp 27692 retention web ui 10.10

### PR DESCRIPTION
**Pull request by guirenard**

### CODE REVIEW

File: elements/directory/l10nvocabulary/nuxeo-l10nvocabulary-edit-layout.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                 label="[[i18n('vocabularyManagement.edit.label_en')]]"
+                 name="label"
+                 value="{{entry.properties.label_en::change}}">
+    </nuxeo-input>
+
+    <nuxeo-input role="widget"
+                 label="[[i18n('vocabularyManagement.edit.label_fr')]]"
+                 name="label"
+                 value="{{entry.properties.label_fr::change}}">
+    </nuxeo-input>
```
Comment (nmpcunha) [inline elements/directory/l10nvocabulary/nuxeo-l10nvocabulary-edit-layout.html]: Is this the expected? English and French?

File: elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html [hunk 2]
```diff
@@ -108,6 +113,27 @@
...
+            <span id="legalHold">[[i18n('documentPage.legalHold')]]</span>
+          </template>
+          <template is="dom-if" if="[[document.retainUntil]]">
+            <iron-icon icon="nuxeo:retain"></iron-icon>
+            <template is="dom-if" if="[[!isRetentionDateIndeterminate(document)]]">
+              <span id="retention" hidden="[[document.hasLegalHold]]">[[_computeRetentionUntiLabel(document)]]</span>
+            </template>
+            <template is="dom-if" if="[[isRetentionDateIndeterminate(document)]]">
+              <span id="indeterminateRetention" hidden="[[document.hasLegalHold]]">[[i18n('documentPage.retainIndeterminate')]]</span>
+            </template>
```
Comment (nmpcunha) [inline elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html]: If I understand this correctly, we'll always have a span and, despite the id, only the text content changes... Should we pass the conditional logic for the computation method instead?

File: elements/directory/l10nvocabulary/nuxeo-l10nvocabulary-edit-layout.html (hunk 1 above)
Comment (Gabez0r) [reply elements/directory/l10nvocabulary/nuxeo-l10nvocabulary-edit-layout.html]: Yes, l10n vobacularies come with translated en and fr labels.

File: elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html (hunk 2 above)
Comment (Gabez0r) [reply elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html]: We could, but I know the distinct id's are used in the tests to distinguis both cases more easily. Alternative here is to either compute a class or check a label, with a latter being something we should avoid...

**Pull request merged**